kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java

```diff
@@ -23,10 +24,10 @@ public SchemaEditView selectSchemaTypeFromDropdown(SchemaCreateView.SchemaType s
         return this;
     }
     @Step
-    public SchemaEditView selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum level) {
+    public static SchemaEditView selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum level) {
         $x("//ul[@name='compatibilityLevel']").click();
         $x("//li[text()='" + level.getValue() + "']").click();
-        return this;
+        return new SchemaEditView();
```

Comment:
no need to do this way. you can call method from this class using 'new SchemaEditView()'

Reply:
Done)